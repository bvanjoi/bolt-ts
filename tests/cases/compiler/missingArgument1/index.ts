// From `github.com/microsoft/TypeScript/blob/v5.7.3/tests/cases/compiler/missingArgument1.ts`, Apache-2.0 License

foo(a,,b);
//~^ ERROR: Argument expression expected.
//~| ERROR: Cannot find name 'foo'.
//~| ERROR: Cannot find name 'a'.
//~| ERROR: Cannot find name 'b'.
