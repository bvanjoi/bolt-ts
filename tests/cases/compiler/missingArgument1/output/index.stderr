  × Argument expression expected.
   ╭─[../../tests/cases/compiler/missingArgument1/index.ts:3:7]
 3 │ ,
   · ─
   ╰────

  × Cannot find name 'foo'.
   ╭─[../../tests/cases/compiler/missingArgument1/index.ts:3:1]
 3 │ foo
   · ───
   ╰────

  × Cannot find name 'a'.
   ╭─[../../tests/cases/compiler/missingArgument1/index.ts:3:5]
 3 │ a
   · ─
   ╰────

  × Cannot find name 'b'.
   ╭─[../../tests/cases/compiler/missingArgument1/index.ts:3:8]
 3 │ b
   · ─
   ╰────
